### PR TITLE
Bug 2061567: add details on updating vSphere credentials

### PR DESCRIPTION
**Pull request by rvanderp3**
The intention of this PR is to provide a procedure describing how to update the vSphere cloud provider credentials.

cc: @jcpowermac @bostrt @GroceryBoyJr

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
/lgtm 
Target: 4.11+
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2061567
Detailed Preview Link: https://deploy-preview-42916--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#rotating_vsphere_credentials_post-install-cluster-tasks

File: modules/rotating-vsphere-credentials.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+====
+If the xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc[vSphere CSI Driver Operator] is enabled, this step is not required.
+====
++
+[source,terminal]
+----
+oc patch kubecontrollermanager cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date )"'"}}' --type=merge
+----
++
+The status of the `kube-controller-manager` operator will report `Progressing=true` while the credentials are rolling out.  The status can be verified by running:
```
Comment (tmalove) [inline modules/rotating-vsphere-credentials.adoc:45]: Close one of the spaces between 'out' and 'The'.

**Review by GroceryBoyJr — COMMENTED** [review]
/lgtm - Richard removed the space. Standing by for QE.

**Comment by jinyunma** [comment]
LGTM

**Comment by jeana-redhat** [comment]
Talked to Richard about this - there is something weird going on with another module not showing up here and I have a few other things that should happen on this one as well. I'm going to pick it up as a new PR to complete the work to avoid a lot of back-and-forth and docs troubleshooting :+1: 

Will close once I have the new one up.

**Comment by jeana-redhat** [comment]
Closing in favor of  #45998

**Pull request closed without merging**